docs/developer/prebuilt_world_loading.md

```diff
@@ -1,94 +1,126 @@
-Loading pre-built Worlds.
-=========================
+<h1>  Loading Pre-Built Worlds </h1>
 
-Aim: To load pre-built Minecraft worlds into the XML missions. 
+**Pre-requisites: Make sure you are able to run Minecraft and the default mission before attempting this tutorial**
 
-Purpose: The pre-built world can have structures and command blocks that define and control various aspects of the mission which we won’t need to code in XML anymore.
+**Aim:** To load pre-built Minecraft worlds into the XML missions </h3> 
 
-Getting Started:
+**Purpose:**  The pre-built world can have hand made structures which will help speed up mission development by shifting some load away from the XML. It is easier to build more complex environments by hand than through the XML. It will also allow us to set environment variables and command blocks in the world and control some aspects of the mission.
 
-    1. Download the Tomcat_Preloaded world from this google drive link: 
 
-https://drive.google.com/drive/folders/1sA5F7ifgzglJnxFaeNuXHFwdMjmrpNHB?usp=sharing
+**Getting Started:**
 
-	and save it somewhere convenient. Unzip the file. Henceforth Tomcat_Preloaded refers
-	to the folder that you got from unzipping.
-	**NOTE: Ensure the path to where you save this folder has no white spaces.
-	Ex:  Documents/folder name is not allowed but Documents/folder_name is allowed**
+1\. Download the SAR_v1 world from this Drive link: 
 
-    2. Download the file_gen and file_gen_skeleton XML s from this google drive link:
+    https://drive.google.com/open?id=19PjgHmUeJmOQ4DS4dAy-4bgax2_TnJRp
 
-https://drive.google.com/drive/folders/1sA5F7ifgzglJnxFaeNuXHFwdMjmrpNHB?usp=sharing
+Save it somewhere convenient and unzip the file. 
 
-	file_gen will serve as a reference for you to use file_gen_skeleton to get your pre-built world 	working. DO NOT SAVE IT INSIDE Tomcat_Preloaded
-	 I saved mine in: 
-	/home/adi-ua/Documents/TOMCAT/tomcat/external/malmo/sample_missions
+Henceforth, SAR_v1 refers to the folder you got from unzipping.
+	
+**NOTE** : Ensure the path to where you save this folder has no white spaces.
+
+Ex:  Documents/folder name is not allowed, but Documents/folder_name is allowed
```

Comment:
It's common practice to render names of files, folders, functions, classes, commands in monospaced/fixed-width fonts in documentation. In order to render some inline text in a fixed-width font in the rendered HTML, simply wrap it with backticks (``).

Things like 'Documents/folder_name', 'SAR_v1', 'file_gen_skeleton', etc. are good candidates for this.